### PR TITLE
DOPE-178: removed bucket and collection fields

### PR DESCRIPTION
**Pull request by jansigi**

### CODE REVIEW

File: core/src/test/kotlin/ch/ergon/dope/TestObjects.kt
Comment (pgruntz) [inline core/src/test/kotlin/ch/ergon/dope/TestObjects.kt:1]: Rename this file to 'Builder' and move it into the helper-package

File: core/src/test/kotlin/ch/ergon/dope/TestObjects.kt [hunk 1]
```diff
@@ -1,79 +1,22 @@
...
-    val geo = addField<StringType>("geo")
-    val alt = addField<StringType>("alt")
-    val lat = addField<StringType>("lat")
-    val lon = addField<StringType>("lon")
-    val icao = addField<StringType>("icao")
-    val id = addField<StringType>("id")
-    val type = addField<StringType>("type")
-    val tz = addField<StringType>("tz")
-}
+fun someStringField(name: String = "strField", bucket: Bucket = UnaliasedBucket("")) = Field<StringType>(name, getBucketName(bucket))
```
Comment (pgruntz) [inline core/src/test/kotlin/ch/ergon/dope/TestObjects.kt:15]: Would really never go for such abbreviations. Better use "numberField", "stringField" and "booleanField"

**Review by pgruntz — APPROVED** [review]

File: core/src/test/kotlin/ch/ergon/dope/TestObjects.kt (hunk 1 above)
Comment (jansigi) [reply core/src/test/kotlin/ch/ergon/dope/TestObjects.kt:15]: done

File: core/src/test/kotlin/ch/ergon/dope/TestObjects.kt
Comment (jansigi) [reply core/src/test/kotlin/ch/ergon/dope/TestObjects.kt:1]: done

**Pull request merged**